docs/source/_static/custom.js

```diff
@@ -31,6 +31,7 @@ document.addEventListener('DOMContentLoaded', () => {
         { selector: '.toctree-l1 > a', text: 'Pixtral (Mistral AI)' },
         { selector: '.toctree-l1 > a', text: 'Many Parallel Jobs' },
         { selector: '.toctree-l1 > a', text: 'Reserved, Capacity Blocks, DWS' },
+        { selector: '.toctree-l1 > a', text: 'Llama-3.2 (Meta)' },
```

Comment:
Add admin policy to new too?

Reply:
Added in #3968 . Let's keep this PR clean : )